### PR TITLE
feat/customWarnMixedMessage

### PR DESCRIPTION
**Pull request by killenb**
- fix: Missing assignment operator in EventMessageAggregator
- chore: increase minor version
- feat: custom warn-mixed message in StatusAggregator
- fix: coding style

### CODE REVIEW

**Comment by mhier** [comment]
Apparently you did not yet merge the other PR, hence its commits are appearing here again...?

**Comment by mhier** [comment]
I would prefer if the custom message were settable also through the constructor (with an optional argument). In many cases the owning ModuleGroup of the aggregator might not have a constructor body, so we would have to add one just for setting the message.

**Review by mhier — APPROVED** [review]

**Review by mhier — CHANGES_REQUESTED** [review]
Please add a test case

**Comment by killenb** [comment]
Argh, sorry. I did not see that you had requested changes. Already force-pushed the merge with the other PR to the master. The last commit (fixup) has the difference wrt. your last review.

File: Modules/src/UserInputValidator.cc [hunk 1]
```diff
@@ -72,9 +72,17 @@ namespace ChimeraTK {
     }
 
     for(auto& accessor : _module->getAccessorListRecursive()) {
-      if(accessor.getDirection() == VariableDirection{VariableDirection::feeding, true} &&
-          accessor.getModel().getTags().count(std::string(UserInputValidator::tagValidatedVariable)) > 0) {
-        _downstreamInvalidatingReturnChannels.emplace(accessor.getAppAccessorNoType().getId());
+      std::cout << "Evaluating accessor " << _module->getName() + accessor.getName() << std::endl;
```
Comment (mhier) [inline Modules/src/UserInputValidator.cc:75]: This looks like forgotten debug output!

**Review by mhier — APPROVED** [review]
Remove forgotten debug output, otherwise OK.

File: Modules/src/UserInputValidator.cc (hunk 1 above)
Comment (killenb) [reply Modules/src/UserInputValidator.cc:75]: Argh, that file should have been stashed.

Comment (killenb) [reply Modules/src/UserInputValidator.cc]: reverted it to the version in the master

**Pull request merged**